api/alerts.go

```diff
@@ -142,29 +147,9 @@ func (a *Alerts) Add(viewName string, newAlert *Alert) (*Alert, error) {
 		"enabled":            graphql.Boolean(newAlert.Enabled),
 		"actions":            actions,
 		"labels":             labels,
+		"runAsUserId":        runAsUserIdPtr,
 	}
-
-	err := a.client.Mutate(&mutation, variables)
-	if err != nil {
-		return nil, err
-	}
-
-	alert = Alert{
-		ID:                 mutation.Alert.ID,
-		Name:               mutation.Alert.Name,
-		QueryString:        mutation.Alert.QueryString,
-		QueryStart:         mutation.Alert.QueryStart,
-		ThrottleField:      mutation.Alert.ThrottleField,
-		TimeOfLastTrigger:  mutation.Alert.TimeOfLastTrigger,
-		IsStarred:          mutation.Alert.IsStarred,
-		Description:        mutation.Alert.Description,
-		ThrottleTimeMillis: mutation.Alert.ThrottleTimeMillis,
-		Enabled:            mutation.Alert.Enabled,
-		Actions:            mutation.Alert.Actions,
-		Labels:             mutation.Alert.Labels,
-		LastError:          mutation.Alert.LastError,
-	}
-	return &alert, nil
```

Comment:
&alert is not used in the output

Reply:
Maybe not in the `humioctl` CLI tool itself, but it is somewhat used within the operator code base: https://github.com/humio/humio-operator/blob/master/pkg/humio/client.go#L589-L593

Changing anything in the api package/dir does influence other things building on top of it, so we should be careful about changing the behavior of the functions. This time it seems like a quick small change and something that is easily updated elsewhere to work with.